client/js/models/archive.js

```diff
@@ -12,5 +17,15 @@ module.exports = {
         file: null
       }
     }
+  },
+  effects: {
+    new: function (data, state, send, done) {
+      window.alert('this should create a new archive')
+      setTimeout(() => done(), 1000)
+    },
+    load: function (data, state, send, done) {
+      window.alert('this should load the archive', data)
+      setTimeout(() => done(), 1000)
+    }
```

Comment:
this is for the importButton's callback.  we'll probably just call send.location(`/${key}`) once the datlink is validated